client/my-sites/plans/current-plan/jetpack-checklist/index.tsx

```diff
@@ -180,25 +225,34 @@ class JetpackChecklist extends PureComponent< Props > {
 					{ isPaidPlan && isRewindUnavailable && productInstallStatus && (
 						<Task
 							{ ...JETPACK_CHECKLIST_TASK_BACKUPS_VAULTPRESS }
+							id="jetpack_vaultpress"
+							onTaskTitleClick={ this.onTaskTitleClick }
+							selectedTaskId={ this.state.selectedTaskId }
 							completed={ vaultpressFinished }
 							href={ JETPACK_CHECKLIST_TASK_BACKUPS_VAULTPRESS.getUrl( siteSlug ) }
+							firstIncomplete={ firstIncomplete }
 							inProgress={ ! vaultpressFinished }
 							onClick={ this.handleTaskStart( { taskId: 'jetpack_backups' } ) }
 						/>
 					) }
 					{ isPaidPlan && productInstallStatus && (
 						<Task
 							{ ...JETPACK_CHECKLIST_TASK_AKISMET }
+							id="jetpack_akismet"
+							onTaskTitleClick={ this.onTaskTitleClick }
+							selectedTaskId={ this.state.selectedTaskId }
 							completed={ akismetFinished }
 							href={ JETPACK_CHECKLIST_TASK_AKISMET.getUrl( siteSlug ) }
+							firstIncomplete={ firstIncomplete }
 							inProgress={ ! akismetFinished }
 							onClick={ this.handleTaskStart( { taskId: 'jetpack_spam_filtering' } ) }
 							target="_blank"
 						/>
 					) }
-					{ this.renderTaskSet( JETPACK_SECURITY_CHECKLIST_TASKS ) }
+
+					{ this.renderTaskSet( JETPACK_SECURITY_CHECKLIST_TASKS, firstIncomplete ) }
```

Comment:
Maybe I messed this up when rebasing, but [`renderTaskSet`](https://github.com/Automattic/wp-calypso/pull/33666/files#diff-109d16ece1aaa451541aafc75a84baf4R99) currently doesn't seem to accept a second arg.